content/en/docs/ops/integrations/spire/index.md

```diff
@@ -151,36 +153,107 @@ Istio will become the Envoy SDS listener if the socket is not created by SPIRE b
 
-1. Use [sidecar injection](/docs/setup/additional-setup/sidecar-injection) to inject the `istio-proxy`
-    container into the pods within your mesh. See [custom templates](/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental)
-    for information on how to apply the custom defined `spire` template into `istio-proxy`. This allows for the CSI driver to mount the UDS on the sidecars.
+Check Ingress-gateway pod state:
 
-    Check Ingress-gateway pod state:
-
-    {{< text bash >}}
-    $ kubectl get pods -n istio-system
-    NAME                                    READY   STATUS    RESTARTS   AGE
-    istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          17s
-    istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
-    {{< /text >}}
+{{< text bash >}}
+$ kubectl get pods -n istio-system
+NAME                                    READY   STATUS    RESTARTS   AGE
+istio-ingressgateway-5b45864fd4-lgrxs   0/1     Running   0          17s
+istiod-989f54d9c-sg7sn                  1/1     Running   0          23s
+{{< /text >}}
 
-Data plane containers will only reach `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
+The Ingress-gateway pod and data plane containers will only reach `Ready` if a corresponding registration entry is created for them on the SPIRE Server. Then,
 Envoy will be able to fetch cryptographic identities from SPIRE.
 See [Register workloads](#register-workloads) to register entries for services in your mesh.
 
 ## Register workloads
 
 This section describes the options available for registering workloads in a SPIRE Server.
 
-### Option 1: Automatic registration using the SPIRE workload registrar
+### Option 1: Registration using the SPIRE Controller Manager
+
+By deploying [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager)
+along with a SPIRE Server, new entries can be automatically registered for each new pod that matches the selector defined in a [ClusterSPIFFEID](https://github.com/spiffe/spire-controller-manager/blob/main/docs/clusterspiffeid-crd.md) custom resource.
 
-By deploying [SPIRE Kubernetes Workload Registrar](https://github.com/spiffe/spire/tree/main/support/k8s/k8s-workload-registrar)
-along with a SPIRE Server, new entries are automatically registered for each new pod that is created.
+1. Create an example ClusterSPIFFEID:
+
+    {{< text bash >}}
+    $ kubectl apply -f - <<EOF
+    apiVersion: spire.spiffe.io/v1alpha1
+    kind: ClusterSPIFFEID
+    metadata:
+      name: example
+    spec:
+      spiffeIDTemplate: "spiffe://{{ .TrustDomain }}/ns/{{ .PodMeta.Namespace }}/sa/{{ .PodSpec.ServiceAccountName }}"
+      podSelector:
+        matchLabels:
+          spiffe.io/spiffe-id: "true"
```

Comment:
```suggestion
          spiffe.io/spiffe-id: "true"
```

xpost nit from the other PR: can we change the label used to e.g.

spiffe.io/spire-managed-identity: "true"

to make it clearer this is specifically saying that SPIRE (not Istio) is attesting and managing the workload identity - the SPIFFE ID is incidental.

Typically you want to use SPIRE so you can actually do granular workload identification beyond what Istio/Kube care about, and would create a ClusterSPIFFEID with specific attestors for _each_ workload you wanted to do granular attestation on (even if they are forced to share the same SPIFFE ID format in Istio, due to Istio being special).

Reply:
Updated!